operations/mimir-mixin/alerts/alerts.libsonnet

```diff
@@ -221,7 +221,26 @@
             message: '{{ $labels.job }}/%(alert_instance_variable)s has a number of mmap-ed areas close to the limit.' % $._config,
           },
         },
-      ] + [
+      ] + (
+        if $._config.forwarding_enabled then [
```

Comment:
No need to be conditional. If forwarding is not enabled, then `cortex_distributor_forward_errors_total` won't be tracked, so the alert can't fire. We adopt the same approach with other optional components to keep the alerts more simple (trying to avoid conditional alerts as much as possible).

Reply:
nice, that simplifies it: f244ead2c6b24c8f06c869b61c1b4c5989964fae